app/graphql/types/query_type.rb

```diff
@@ -30,5 +30,13 @@ def allCandidates
     def candidate(id:)
       return Candidate.find(id)
     end
+
+    field :jobApplications, [Types::JobApplicationType], null: false do
+      description "This will get all of the job applications."
```

Comment:
Added a new field to be displayed as an array of job applications when the query is completed.